extensions/interactions/ItemSelectionInput/directives/ItemSelectionInputValidationServiceSpec.js

```diff
@@ -191,4 +225,42 @@ describe('ItemSelectionInputValidationService', function() {
           'rule 1, the "proper subset" rule must include at least 2 options.')
       }]);
     });
+
+  it(
+    'should expect number of correct options to be in between the maximum ' +
+    'and minimum allowed selections when the "Equals" rule is used.',
+    function() {
+      // Make min allowed selections greater than correct answers.
+      customizationArguments.minAllowableSelectionCount.value = 2;
+
+      var warnings = validatorService.getAllWarnings(
+        currentState, customizationArguments, OneInputAnswerGroups,
+        goodDefaultOutcome);
+      expect(warnings).toEqual([{
+        type: WARNING_TYPES.ERROR,
+        message: (
+          'In answer group 1, ' +
+          'rule 1, the "Equals" rule\'s correct options should be between ' +
+          'the minimum and maximum allowed selection count.')
+      }]);
+    });
+
+  it(
+    'should expect at least one option when  ' +
+    '"ContainsAtLeastOneOf" rule is used.',
+    function() {
+      // Make min allowed selections greater than correct answers
```

Comment:
Add period(.) at the end of the comment.

Reply:
>This error message is still wrong. Have you understood this part of the code you've written? It raises warning only when there is no selected choice (i.e. at least one choice should be selected).

I am so sorry it slipped through, I was checking for equals function, And I may have modified this by mistake (I still don't know how, changed it now). And yeah I completely understood the code I wrote  (I mentioned this check in the comment in the issue #6418 ).

Reply:
Done, Made all the changes, please take a look @vibhor98 
And yeah I think I got it, this is what happened.
I copied the check for Equals and modified it to make check for ContainsAtLeastOneOf, And then after modifying I forgot to save, and then rebooted, and Idk how vscode didnt retain those changes. (this happened very often, I will be careful from now, I have always used vim, Switching to vscode now, and Ctrl+S is the command that sometimes permanently freezes vim :L )